examples/demo.yaml

```diff
@@ -60,6 +60,12 @@ spec:
       containers:
       - name: nginx
         image: nginx
+        livenessProbe:
```

Comment:
Why add this livenessProbe?

Reply:
I think it is helpful for local testing to ensure a disruption does not (or does if this is what's expected) disrupt liveness probes. A concrete example is applying a 100% drop network disruption on a pod should not drop packets sent for probing.